File: JAVA/src/test/FizzBuzzTest.java
import org.junit.Assert;
import org.junit.Test;

public class FizzBuzzTest {

    @Test
    public void FizzBuzzConverterLeavesNormalNumbersAlone() {

        FizzBuzzConverter fizzBuzz = new FizzBuzzConverter();

        Assert.assertEquals("1", fizzBuzz.convert(1));
    }

    @Test
    public void FizzBuzzConverterForMultiplesOfThree() {

        FizzBuzzConverter fizzBuzz = new FizzBuzzConverter();

        Assert.assertEquals("Fizz", fizzBuzz.convert(3));
    }

    @Test
    public void FizzBuzzConverterForMultiplesOfFive() {

        FizzBuzzConverter fizzBuzz = new FizzBuzzConverter();

        Assert.assertEquals("Buzz", fizzBuzz.convert(5));
    }

    @Test
    public void FizzBuzzConverterForMultiplesOfFifteen() {

        FizzBuzzConverter fizzBuzz = new FizzBuzzConverter();

        Assert.assertEquals("FizzBuzz", fizzBuzz.convert(15));
    }

}

File: SCALA/fizzbuzzscala/src/test/scala/FizzBuzzTest.scala
class FizzBuzzTest extends org.scalatest.FunSuite {
  test("Returns FizzBuzz when multiple of 15") {
    assert(FizzBuzz.FizzBuzzConverter(15) === ("FizzBuzz"))
  }
  test("Returns Buzz when multiple of 5") {
    assert(FizzBuzz.FizzBuzzConverter(5) === ("Buzz"))
  }
  test("Returns Fizz when multiple of 3") {
    assert(FizzBuzz.FizzBuzzConverter(3) === ("Fizz"))
  }
}
